SpatialGDK/Documentation/toc.md

```diff
@@ -37,6 +37,7 @@
         - [Non-Unreal server-worker types]({{urlRoot}}/content/non-unreal-server-worker-types)
     - [Snapshots]({{urlRoot}}/content/spatialos-concepts/generating-a-snapshot) 
     - [Gameplay Ability System]({{urlRoot}}/content/ability-system)
+    - [Command line arguments]({{urlRoot}}/content/command-line-arguments)
```

Comment:
I think this page should go with Helper scripts, Directory structure as it's about Workflows and the mechanics of using the GDK.